docs/api/providers/nodebb/notification.md

```diff
@@ -53,7 +54,7 @@ Create a Notification class and bind it to a forum instance
 #### bindNotification~notificationType
 Notification types enum
 
-**Kind**: inner constant of <code>[bindNotification](#sockbot.providers.nodebb.module_Notification.bindNotification)</code>  
```

Comment:
huh. interesting. looks like something you did changed the detection of notificationType from constant to enum (which is more correct actually)

i wouldn't worry about it.